monitoring/lifecycle/alerts.yaml

```diff
@@ -181,3 +181,23 @@ groups:
       description: "More than 5% of Kafka messages failed to publish to the lifecycle object topic"
       summary: "High rate of failed messages to the object topic"
 
+  - alert: KafkaConsumerSlowTask
+    Expr: |
+      sum(s3_zenko_queue_slowTasks_count{namespace="${namespace}"}) by(job) > 0
+    For: "5m"
+    Labels:
+     severity: warning
+    Annotations:
+      description: Some tasks are taking too long to process in {{ $labels.job }}.
+      summary: "Kafka message processing time too high"
+
+  - alert: KafkaConsumerRebalanceTimeout
+    Expr: |
+      sum(s3_zenko_queue_rebalance_total_count{namespace="${namespace}", status="timeout"}) by(pod) > 0
+    Labels:
+     severity: critical
+    Annotations:
+      description: >-
+        Kafka rebalance has timed out for pod `{{ $labels.pod }}`, which indicates that the consumer
```

Comment:
i still have some doubt that there may be corner-cases where a delay is appropriate, so i will stick with `should` :-)

anyway users _should_ (😉) never this see, since the process is expected to become unhealthy when this happens...